components/board/server_test.go

```diff
@@ -16,8 +16,8 @@ import (
 )
 
 var (
-	errFoo           = errors.New("whoops")
-	errUnimplemented = errors.New("not found")
+	errFoo      = errors.New("whoops")
+	errNotFound = errors.New("not found")
```

Comment:
just changed this name to reflect the error better since the current name confused me.